kafka/kafkatest/utils.go

```diff
@@ -108,6 +111,7 @@ func (f *FakeKafkaConsumer) CommitMessage(msg *kafkalib.Message) error {
 	f.log.WithField("offset", msg.TopicPartition.Offset).Trace("commiting message...")
 	if int64(msg.TopicPartition.Offset) > f.offset {
 		f.offset = int64(msg.TopicPartition.Offset)
+		f.uncommitedOffset = f.offset
```

Comment:
Would this cause the reader to jump back to this msg then? It feels like this could loop on the same msg

Reply:
Oh good point